### PR TITLE
Ext/create button

### LINKED ISSUE #1544: Extensibility: Expand Pizza example to cover all the components we have
**Description**

Even with comprehensive documentation, sometimes an example is 10x better. Our PIZZA Example should showcase all our widgets and also if possible all the variations that they support.

**Reasons**

Users find it easier to get the right configuration from an working example

**Acceptance Criteria**

- [x] Our pizza CRD has enough fields and options to cover our components
- [x] The example provided in the github contains all our available components for list, detail and edit forms
- [x] We move our extensibility examples into it's own extensibility folder inside examples
- [ ] We will use kube-public for our examples

### LINKED ISSUE #1554: Extensibility: List View: Specify that the create button should be disabled
**Description**
Allow the possibility to disable the Create button for certain custom resources.

**Reasons**
We have a resource that serves as a report from the work that our operator has done. As such the user has no business creating an instance of that resource. This is very similar to how you can not create Events from the Kyma Dashboard either.

### PR DESCRIPTION
**Pull request by Lyczeq**
**Description**

Changes proposed in this pull request:

- Added the `disableCreate` flag to the extensibility - resource which is a boolean set by default to false. If set to true it doesn't allow hiding the Create button on the list view
- Added the `disableCreate` propType to the component
- Improved docs

**Related issue(s)**
Resolves #1554

### CODE REVIEW

File: docs/extensibility/resources.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ The `resource` section is required and contains basic information about the reso
 - **path** - path fragment for this resource used in the URL. Defaults to pluralized lowercase **kind**. Used to provide an alternative URL to avoid conflicts with other resources.
 - **defaultPlaceholder** - to be shown in place of empty resource leaves. Overridden by the widget-level **placeholder**. Defaults to `-`.
 - **description** - displays a custom description on the resource list page. It can contain links. If the translation section has a translation entry with the ID that is the same as the **description** string, the translation is used.
+- **allowCreate** - either `true` or `false`. Defaults to `true`
```
Comment (grego952) [inline docs/extensibility/resources.md:19]:
```suggestion
- **allowCreate** - either `true` or `false`. Defaults to `true`.
```

File: core-ui/src/components/Extensibility/ExtensibilityList.js [hunk 2]
```diff
@@ -62,6 +62,7 @@ export const ExtensibilityListCore = ({ resMetaData }) => {
     <ResourcesList
       createResourceForm={ExtensibilityCreate}
       allowSlashShortcut
+      disableCreate={allowCreate === undefined ? false : !allowCreate}
```
Comment (Wawrzyn321) [inline core-ui/src/components/Extensibility/ExtensibilityList.js:65]:
```suggestion
      disableCreate={allowCreate !== false}
```

**Review by Wawrzyn321 — COMMENTED** [review]
I think it works good. However I'd leave the discussion between `allowCreate` and possible `disableCreate` in configmap.

**Review by grego952 — DISMISSED** [review]

**Review by Wawrzyn321 — APPROVED** [review]

**Review by grego952 — DISMISSED** [review]

**Review by Wawrzyn321 — APPROVED** [review]

**Review by Wawrzyn321 — APPROVED** [review]

**Review by grego952 — APPROVED** [review]

**Pull request merged**